tests/lists.rs

```diff
@@ -1,73 +1,76 @@
-use crate::loaders::*;
-use hvm_lang::term::DefinitionBook;
-use insta::assert_debug_snapshot;
-
 mod loaders;
 
-fn list_got(index: u32) -> DefinitionBook {
-  let template = load_file("list_put_got.hvm");
-  let code = replace_template(template, &[("{fun}", "got"), ("{args}", &index.to_string())]);
-  parse_lang(&code)
-}
+#[cfg(not(feature = "cuda"))] //Cuda does not support native numbers
+mod list_tests {
```

Comment:
Cuda doesn't support native numbers, but why do the list tests need numbers? It's because it was a simple value that we put in the lists, right? Can't we change those values to be something else instead, so we can use these tests for the GPU as well?